.github/workflows/triage-move-review-requests.yml

```diff
@@ -77,15 +77,15 @@ jobs:
     name: Move PRs asking for product review to the product board
     runs-on: ubuntu-latest
     # Skip in forks
-    if: github.repository == 'vector-im/element-android'
+    if: github.repository == 'element-hq/element-android'
     steps:
       - uses: octokit/graphql-action@v2.x
         id: find_team_members
         with:
           headers: '{"GraphQL-Features": "projects_next_graphql"}'
           query: |
             query find_team_members($team: String!) {
-              organization(login: "vector-im") {
+              organization(login: "element-hq") {
```

Comment:
Not sure about this occurrence and the one above, but I guess it will be fine?

Reply:
Only one way to find out 😅 .